dev/new-rule/templates/doc.md

```diff
@@ -0,0 +1,53 @@
+# (rule name goes here)
```

Comment:
I just updated this template today, can you rebase on the latest master and use my updated version?